src/Xamarin.Android.Build.Tasks/Tasks/MakeBundleNativeCodeExternal.cs

```diff
@@ -66,6 +70,20 @@ public override bool Execute ()
 			return !Log.HasLoggedErrors;
 		}
 
+		private static string ParseI18nFlags (string i18n)
```

Comment:
I think this method is working *way* too hard.

[`$(MandroidI18n)` can contain](https://developer.xamarin.com/guides/android/under_the_hood/build_process/):

* None
* All
* CJK
* MidEast
* Other
* Rare

This should align ~directly with the values that `mkbundle --i18n` permits. That said, *casing* is a concern; Xamarin.Android uses `None`, while `mkbundle` uses `none`

I thus think we could *try* (entirely untested!):

    static Dictionary<string, string> I18nMappings = new Dictionary<string, string> (StringComparer.OrdinalIgnoreCase) {
        { "None", "none" }.
        { "All", "all" },
        { "CJK", "CJK" },
        { "MidEast", "MidEast" },
        { "Other", "Other" },
        { "Rare", "Rare",
    };
    static string ParseI18nFlags (string i18n)
    {
        var mkbundle = new List<string> ();
        var values = (i18n ?? "").Split (',', ';', ':');
        foreach (var k in values) {
            string v;
            if (I18nMappings.TryGetValue (k, out v))
                mkbundle.Add (v);
           else
                Log.LogCodedError (...);
        }
        return string.Join (",", mkbundle);
    }

The above *should* map *any* "input" casing of `$(MandroidI18n)` to the correct "output", e.g. if someone has `<MandroidI18n>MIDEAST</MandroidI18n>`, we'll correctly transform that to `mkbundle ... --i18n MidEast`.